lib/API.jsx

```diff
@@ -762,10 +762,12 @@ export default function API(network, args) {
 
         card: {
             /**
-             * Set the limit of an expensify card
+             * Set the limit of an expensify card. We need to pass either `email` or `cardUserEmail`.
+             * `cardUserEmail` is correct, but email is still supported across the rollout to `cardUserEmail`.
```

Comment:
Let's have a followup GH to clean this up after it's released?

Reply:
Yes, I'll make that.